src/classes/EscalationTrigger_Test.cls

```diff
@@ -0,0 +1,190 @@
+/**
+* Test for the escalation trigger
+*
+* @author Preeti (ppreeti@redhat.com)
+*/
+
+@IsTest
+public class EscalationTrigger_Test {
+	////////////////////////////////////////////////////////////////////////////////////
+	//                             setAccountOnInsert                                 //
+	////////////////////////////////////////////////////////////////////////////////////
+	static testMethod void setAccountOnInsert_primaryCaseNotNull_valid() {
+		Account testAccount = TestUtils.getAccount();
+		insert testAccount;
+
+		Case testCase = TestUtils.getCase(testAccount);
+		insert testCase;
+
+		rh_escal__Escalation__c testEscalation = TestUtils.getEscalation();
+		testEscalation.rh_escal__PrimaryCase__c = testCase.Id;
+
+		Test.startTest();
+
+		insert testEscalation;
+
+		Test.stopTest();
+
+		testEscalation = TestUtils.fetchEscalation(testEscalation);
+
+		System.assertEquals(testCase.Id, testEscalation.rh_escal__PrimaryCase__c, 'The primary case of escalation should not be null.');
```

Comment:
Change this message to read `The primary case is not set correctly`.  Do this here and throughout the test.